modules/vault-cluster/main.tf

```diff
@@ -132,8 +132,8 @@ module "security_group_rules" {
 
   security_group_id                    = "${aws_security_group.lc_security_group.id}"
   allowed_inbound_cidr_blocks          = ["${var.allowed_inbound_cidr_blocks}"]
-  allowed_inbound_security_group_ids   = ["${var.allowed_inbound_security_group_ids}"]
-  allowed_inbound_security_group_count = "${var.allowed_inbound_security_group_count}"
+  allowed_inbound_security_group_ids   = "${concat(list(aws_security_group.lc_security_group.id), var.allowed_inbound_security_group_ids)}"
```

Comment:
I think the proper way to do this is to add an explicit rule with `self` in the security group rules module.

Also, I could've _sworn_ we did that already...

Reply:
I found my source of confusion: The rule that's there now is for the cluster port (8201). My usecase is expecting the API port (8200) also. I've updated this PR to use the 'self' approach for both. If exposing the API port between nodes doesn't fit your intent (e.g. - minimal permission) then I'll cancel this PR & implement it on my side.

Reply:
Ohhhh... Hm, I think exposing that port between nodes is the right thing to do, thx!